media-libs/jbigkit/jbigkit-2.1.recipe

```diff
@@ -0,0 +1,88 @@
+SUMMARY="A lossless image compression library"
+DESCRIPTION="JBIG-KIT is a software implementation of the JBIG1 data \
+compression standard (ITU-T T.82), which was designed for bi-level image \
+data, such as scanned documents. This library is available in portable C \
+code. It is widely used in fax products, printer firmware, printer drivers, \
+document management systems and imaging software."
+HOMEPAGE="https://www.cl.cam.ac.uk/~mgk25/jbigkit/"
+COPYRIGHT="1995-2014 Markus Kuhn
+	1989, 1991 Jef Poskanzer"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://www.cl.cam.ac.uk/~mgk25/download/jbigkit-$portVersion.tar.gz"
+CHECKSUM_SHA256="de7106b6bfaf495d6865c7dd7ac6ca1381bd12e0d81405ea81e7f2167263d932"
+PATCHES="jbigkit-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86_gcc2 x86"
+
+libVersion=$portVersion
+libVersionCompat="$libVersion compat >= 2"
+
+PROVIDES="
+	jbigkit$secondaryArchSuffix = $portVersion
+	cmd:jbgtopbm$secondaryArchSuffix = $portVersion
+	cmd:jbgtopbm85$secondaryArchSuffix = $portVersion
+	cmd:pbmtojbg$secondaryArchSuffix = $portVersion
+	cmd:pbmtojbg85$secondaryArchSuffix = $portVersion
+	lib:libjbig$secondaryArchSuffix = $libVersionCompat
+	lib:libjbig85$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	jbigkit${secondaryArchSuffix}_devel = $portVersion
+	devel:libjbig$secondaryArchSuffix = $libVersionCompat
+	devel:libjbig85$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	jbigkit$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+defineDebugInfoPackage jbigkit$secondaryArchSuffix \
+	"$libDir"/libjbig.so \
+	"$libDir"/libjbig85.so
+
+BUILD()
+{
+	make all $jobArgs
+	make -C libjbig -f Makefile_so $jobArgs
+}
+
+INSTALL()
+{
+	install -m 0755 -d "$binDir" "$includeDir" "$libDir" \
+		"$developLibDir" "$manDir/man5"
+
+	# install the tools
+	cd pbmtools
+	install -m 755 -t "$binDir" jbgtopbm{,85} pbmtojbg{,85}
+
+	# install man pages
+	install -m 0644 -t "$manDir/man5" pbm.5 pgm.5
+
+	# install headers and libs
+	cd ../libjbig
+	install -m 0644 -t "$includeDir" *.h
+	install -m 0644 -t "$libDir" libjbig*
+
+	prepareInstalledDevelLibs libjbig libjbig85
+
+	packageEntries devel \
+		$developDir
+}
+
+TEST()
+{
+	make -C libjbig test
```

Comment:
```diff
-	make -C libjbig test
+	make test
```
(There's a test target in the main Makefile, and it will call make test in libjbig and then in pbmtools.)
I did not try this, so maybe I should not be making this suggestion :smile:

Reply:
Works, only needed to add one cmd for the test ...